sidekiq-lock.gemspec

```diff
@@ -18,10 +18,4 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "sidekiq", ">= 2.14.0"
   spec.add_dependency "redis",   ">= 3.0.5"
-
-  spec.add_development_dependency "bundler"
-  spec.add_development_dependency "rake"
-  spec.add_development_dependency "mocha", "~> 0.14.0"
-  spec.add_development_dependency "minitest"
-  spec.add_development_dependency "appraisal"
```

Comment:
unsure what's the best practice nowadays, seems like sidekiq itself defines all this stuff in Gemfile now